### PR TITLE
charts

### PR DESCRIPTION
**Pull request by xithan**

### CODE REVIEW

File: mod/01_filter/set/abstract/00_filter_helper.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  args.reverse_merge! paging_hash
+  args[:filter] ||= {}
+  args[:filter].reverse_merge! filter_hash
+  args.reverse_merge! sort_hash
+  args
+end
+
+format do
+  delegate :filter_hash, :sort_hash, :paging_hash, to: :card
+end
```
Comment (ethn) [inline mod/01_filter/set/abstract/00_filter_helper.rb]: flag for next huddle: let's figure out mod / modfile ordering!

(just notice the 00 in the name ;) )

File: mod/01_layout/set/abstract/table.rb [hunk 2]
```diff
@@ -24,10 +24,23 @@ def wikirate_table table_type, row_cards, cell_views, opts={}
...
+    add_td_classes opts, ["header", "data", "details"]
     wikirate_table table_type, item_cards, cell_views, opts
   end
 
+  def add_td_classes opts, new_classes
+    opts[:td] ||= {}
+    opts[:td][:classes] ||= []
+    classes = opts[:td][:classes]
+    new_classes.each_with_index do |cl, i|
+      if classes[i]
```
Comment (ethn) [inline mod/01_layout/set/abstract/table.rb:36]: new `#css_classes` method (used by `add_class`) allows: 
```
classes[i] = css_classes classes[i], cl
```

File: mod/01_metrics/set/type/metric_value/answers.rb [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    ]
+    end
+  end
+
+  def value_field
+    wrap_with :div, class: "value pull-left" do
+    [
+      wrap_with(:span, currency, class: "metric-unit"),
+      _render_value_link,
+      wrap_with(:span, legend, class: "metric-unit"),
```
Comment (ethn) [inline mod/01_metrics/set/type/metric_value/answers.rb]: maybe `with_unit { render_value_link }`?

File: mod/01_metrics/set/type/metric_value/answers.rb [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  end
+
+  view :answer_details do
+    value_details
+  end
+
+  view :answer_details_toggle do
+    css_class = "fa fa-caret-right fa-lg margin-left-10 btn btn-default btn-sm pull-right"
+    wrap_with(:i, "", class: css_class,
+              data: { toggle: "collapse",
```
[the remark this answers is not in the document]
Comment (ethn) [reply mod/01_metrics/set/type/metric_value/answers.rb]: hound has a lot of quick fixes for this file.

File: db/migrate_cards/20161005120800_metric_answer_lookup_table.rb [hunk 5]
```diff
@@ -12,13 +12,14 @@ def up
...
       t.string   :metric_record_name
       t.string   :value
+      t.float    :numeric_value
       t.datetime :updated_at
       t.boolean  :imported
-      t.boolean :latest
+      t.boolean  :latest
     end
 
     add_index :metric_answers, :metric_answer_id,
```
Comment (ethn) [inline db/migrate_cards/20161005120800_metric_answer_lookup_table.rb]: I know it's a bit late to say this, and we probably don't want to do anything right away, but it's been occurring to me that perhaps we should use "Record" "Answer" and "Answer Value" in code.  Just flagging this to discuss before we set about any grand renaming.

**Review by ethn — APPROVED** [review]

File: mod/01_filter/set/abstract/00_filter_helper.rb (hunk 1 above)
Comment (xithan) [reply mod/01_filter/set/abstract/00_filter_helper.rb]: modfile ordering is missing but we have mod ordering.

File: db/migrate_cards/20161005120800_metric_answer_lookup_table.rb (hunk 5 above)
Comment (xithan) [reply db/migrate_cards/20161005120800_metric_answer_lookup_table.rb]: would save a lot of "metric" typing (and reading)

**Pull request merged**